modules/ROOT/pages/_partials/gs-accessing-prometheus-grafana-proc.adoc

```diff
@@ -0,0 +1,18 @@
+[id='gs-accessing-prometheus-grafana-proc']
+
+ifdef::env-github[]
+:imagesdir: ../images/
+endif::[]
+
+= Accessing Prometheus and Grafana
+
+.Prerequisite
+* You have logged in to the OpenShift console as the `customer-admin` user.
+
+.Procedure
+. In the *My Projects* section, click *View All*.
+. Select the *Managed Service Monitoring* project.
+. From the sidebar, select *Applications* -> *Routes*.
```

Comment:
Just an fyi, asciidoctor provides a macro for menu selections. So instead of this, you could do menu:Applications[Routes].
https://asciidoctor.org/docs/user-manual/#menu-selections

Reply:
Thanks, I've done some experimenting with that macro. It only works is the experimental attribute is set. 

https://asciidoctor.org/docs/user-manual/#user-interface-macros

Do we know if this attribute is set on the live docs?